code/datums/mutations/touch.dm

```diff
@@ -37,15 +37,19 @@
 			C.dropItemToGround(C.get_inactive_held_item())
 			C.confused += 15
 			C.visible_message("<span class='danger'>[user] electrocutes [target]!</span>","<span class='userdanger'>[user] electrocutes you!</span>")
+			use_charge(user)
 			return ..()
 		else
 			user.visible_message("<span class='warning'>[user] fails to electrocute [target]!</span>")
+			use_charge(user)
```

Comment:
if proximity is true it will use double the charges by calling the super method

Reply:
That doesn't seem to do anything? Same cooldown, no runtimes
![image](https://user-images.githubusercontent.com/110184118/225705697-b211f9f3-2902-4b0c-b6b1-d436d389dcb8.png)